src/vs/workbench/services/preferences/browser/keybindingsEditorInput.ts

```diff
@@ -44,7 +44,7 @@ export class KeybindingsEditorInput extends EditorInput {
 	}
 
 	override matches(otherInput: EditorInput | IUntypedEditorInput): boolean {
-		return super.matches(otherInput) || otherInput instanceof KeybindingsEditorInput;
```

Comment:
I'm curious why `super.matches` returned true? I assume it would be looking at URI at least?

Reply:
Yeah not sure why, but this returned true `isEqual(this.resource, EditorResourceAccessor.getCanonicalUri(otherInput))` I didn't dig too much into it because we shouldn't be calling super anyways since super is a very loose matches